src/lib/__tests__/constants.test.ts

```diff
@@ -0,0 +1,64 @@
+import { describe, expect, test } from "bun:test"
+import { TSCI_PACKAGE_PATTERN } from "../constants"
+
+describe("TSCI_PACKAGE_PATTERN", () => {
+  // Test valid patterns
+  const validPackages = [
+    "@tsci/basic",
+    "@tsci/package-name",
+    "@tsci/multiple-dashes-here",
+    "@tsci/package.subpackage",
+    "@tsci/complex-package.complex-subpackage",
+    "@tsci/a.b.c",
+    "@tsci/package-with-numbers123",
+    "@tsci/package123-with-numbers",
+    "@tsci/core.utils-helper",
+  ]
+
+  validPackages.forEach((pkg) => {
+    test(`should match valid package: ${pkg}`, () => {
+      const matches = Array.from(pkg.matchAll(TSCI_PACKAGE_PATTERN))
+      expect(matches).toHaveLength(1)
+      expect(matches[0][0]).toBe(pkg)
+    })
+  })
+
+  // Test invalid patterns
+  const invalidPackages = [
+    "@tsci/", // Empty
+    "@tsci/123start-with-number", // Starts with number
+    "@tsci/-start-with-dash", // Starts with dash
+    "@tsci/.start-with-dot", // Starts with dot
+    "@tsci/double--dash", // Double dash
+    "@tsci/double..dot", // Double dot
+    "@tsci/end-with-dash-", // Ends with dash
+    "@tsci/end-with-dot.", // Ends with dot
+    "@tsci/invalid@chars", // Invalid characters
+    "@tsci/dot.dash.-mixed", // Invalid dot-dash combination
+    "@tsci/dash-.dot-mixed", // Invalid dash-dot combination
```

Comment:
This is also acceptable although you dont need to fix it in this PR because its rare